types/office-js-preview/index.d.ts

```diff
@@ -10894,6 +10894,38 @@ declare namespace Office {
          *                   OR a {@link Office.ReplyFormData | ReplyFormData} object that contains body or attachment data and a callback function.
          */
         displayReplyAllForm(formData: string | ReplyFormData): void;
+        /**
+         * Displays a reply form that includes the sender and all recipients of the selected message or the organizer and all attendees of the
+         * selected appointment.
+         *
+         * In Outlook on the web, the reply form is displayed as a pop-out form in the 3-column view and a pop-up form in the 2- or 1-column view.
```

Comment:
I'm not sure the dash in "2-" is needed. I think it's implied in the following "1-column".
```suggestion
         * In Outlook on the web, the reply form is displayed as a pop-out form in the 3-column view and a pop-up form in the 2 or 1-column view.
```

Reply:
@AlexJerabek These are suspended compound modifiers (see [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28765)) but since there's no space limit here, I'll spell them out.